build.assets/Makefile

```diff
@@ -125,7 +125,7 @@ build-binaries-fips: buildbox-centos7-fips webassets
 .PHONY:buildbox
 buildbox:
 	$(call PULL_ON_CI,$(BUILDBOX))
-	DOCKER_BUILDKIT=1 docker build --platform=linux/$(RUNTIME_ARCH) \
+	docker buildx build \
```

Comment:
Have you tested it on all pipelines? I remember that one of them (I think that was ARM on GHA) didn't have buildkit installed. I'm not sure if `buildx` is available there (technically it's the same package but 🤷 )

Reply:
I thought that was one of the drone workers, as I saw something similar. @camscale and I have been testing the pipelines. We haven't seen any buildkit-related issues so far, 🤞.